doc_src/index.hdr.in

```diff
@@ -1068,16 +1070,15 @@ Command mode is also known as normal mode.
 
 - @key{[} and @key{]} search the command history for the previous/next token containing the token under the cursor before the search was started. See the <a href='#history'>history</a> section for more information on history searching.
 
-- @key{Control, X} copies the current buffer to the system's clipboard, @key{Control, V} inserts the clipboard contents.
 - @key{Control,C} deletes the entire line.
 
 - @key{Backspace} moves the cursor left.
 
 \subsubsection vi-mode-insert Insert mode
 
-- @key{Escape} or @key{Control,C} enters <a href="#vi-mode-command">command mode</a>.
+- @key{Escape} enters <a href="#vi-mode-command">command mode</a>.
 
-- @key{Control,x} moves the cursor to the end of the line. If an autosuggestion is available, it will be accepted completely.
+- @key{Control,C} deletes the entire line.
```

Comment:
The description was correct up until we merged commit 5f849d026 to address issue #2904. That changed the [ctrl-C] key binding. Unfortunately this part of the documentation wasn't updated. Your change is an opportunity to fix that oversight.